src/app/pages/preferences/page/forms/general-preferences-form.component.ts

```diff
@@ -158,7 +158,8 @@ export class GeneralPreferencesFormComponent implements OnInit, AfterViewInit, O
           name: 'allowPwToggle',
           placeholder: T('Enable Password Toggle'),
           value:prefs.allowPwToggle,
-          tooltip: T('This option enables/disables a password toggle button.'),
+          tooltip: T('This option enables an icon that can be \
```

Comment:
Still not very clear. Suggestion:
`When set, an "eye" icon appears next to password fields. Clicking the icon reveals the password.`